modules/integration_azure-datafactory/conf/01-adf-activity-run.yaml

```diff
@@ -0,0 +1,22 @@
+module: "Azure DataFactory"
+name: "activity error rate"
+filtering: "filter('resource_type', 'Microsoft.DataFactory/factories') and filter('primary_aggregation_type', 'true')"
+aggregation: ".sum(by=['name'])"
```

Comment:
hum, is this enough? what if you have multiple pipeline with the same name il multiple rg/regions?